20854739 the phps do not build
This fix will not be sent upstream since this version has been EOF
by the community.

--- php-5.3.29/ext/gd/config.m4_orig	2014-08-13 12:22:50.000000000 -0700
+++ php-5.3.29/ext/gd/config.m4	2015-04-15 11:35:17.456859370 -0700
@@ -162,7 +162,7 @@
   if test "$PHP_FREETYPE_DIR" != "no"; then
 
     for i in $PHP_FREETYPE_DIR /usr/local /usr; do
-      if test -f "$i/include/freetype2/freetype/freetype.h"; then
+      if test -f "$i/include/freetype2/freetype.h" || test -f "$i/include/freetype2/freetype/freetype.h"; then
         FREETYPE2_DIR=$i
         FREETYPE2_INC_DIR=$i/include/freetype2
         break
